From fffe5d622549f85968ea0be9717b90cbc020be71 Mon Sep 17 00:00:00 2001
From: Eric Lemanissier <eric.lemanissier@gmail.com>
Date: Mon, 21 Jan 2019 13:14:09 +0100
Subject: [PATCH] configure: handle FREETYPE_* variables

This allows the user to set the following variables:
FREETYPE_INCDIR
FREETYPE_LIBDIR
FREETYPE_LIBS

Fixes: QTBUG-72765
Change-Id: Ib70b30f355cc2185c45233e0cfe5e6e55acdf2b6
Reviewed-by: Joerg Bornemann <joerg.bornemann@qt.io>
---
 src/gui/configure.json | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/gui/configure.json b/src/gui/configure.json
index 0332631ec8..82ca9603b1 100644
--- a/src/gui/configure.json
+++ b/src/gui/configure.json
@@ -239,5 +239,6 @@
             "sources": [
                 { "type": "pkgConfig", "args": "freetype2" },
+                { "libs": "-lfreetype" },
                 { "type": "freetype", "libs": "-lfreetype", "condition": "!config.wasm" },
                 { "libs": "-s USE_FREETYPE=1", "condition": "config.wasm" }
             ],
--
2.18.0.windows.1
